src/logics/ArithLogic.cc

```diff
@@ -1,10 +1,11 @@
 #include "ArithLogic.h"
 
 #include <common/ApiException.h>
-#include <common/FastRational.h>
 #include <common/InternalException.h>
 #include <common/StringConv.h>
 #include <common/TreeOps.h>
+#include <common/numbers/FastRational.h>
+#include <common/numbers/Number.h>
```

Comment:
Do you really need to include the same file both in `.h` and the `.cc` file?
If it is included in the header, I would prefer to not include it again the the implementation file.

Reply:
I will remove `Number.h`.
`FastRational.h` is not necessarily included by `ArithLogic.h` if `FAST_RATIONALS` is not defined, but since it is hardcoded this way it would not work in the other case anyway.
So I will remove it as well ...